tasks/__init__.py

```diff
@@ -53,7 +53,7 @@ def decorator(f):
 
 
 @task
-def server(host=None, port=5000, debug=True, live=False, gitlogs=False):
+def server(host="127.0.0.1", port=5000, debug=True, live=False, gitlogs=False):
```

Comment:
I think that modifications to `tasks/__init__.py` should be a separate PR, or at least, excluded from this PR.  And I'm pretty sure (I'll try to find them) that some of these modifications have already been made in the canonical osf.io repo.